src/libData/AccountData/AccountStoreSC.tpp

```diff
@@ -486,15 +497,19 @@ bool AccountStoreSC<MAP>::ParseCreateContractJsonOutput(
   }
   gasRemained = atoi(_json["gas_remaining"].asString().c_str());
 
-  if (!_json.isMember("message") || !_json.isMember("states")) {
-    LOG_GENERAL(WARNING,
-                "The json output of this contract is corrupted or create "
-                "contract failed");
+  if (!_json.isMember("message") || !_json.isMember("states") ||
```

Comment:
Should it be `&&`?

Reply:
Another one below as well

Reply:
&& is for both of the condition doesn't meet. || here means if any of the condition doesn't meet.